requirements.txt

```diff
@@ -32,3 +32,4 @@ urllib3==2.0.3
 visdcc==0.0.50
 Werkzeug==2.2.3
 zipp==3.15.0
+matplotlib==3.7.3
```

Comment:
Just be sure this version exists on python 3.8, since we said the repo only needs 3.8

Reply:
Compatible!
https://matplotlib.org/3.7.3/devel/dependencies.html

Reply:
Also, there is github action for python3.8.